source/mesh_deformation/fastscape.cc

```diff
@@ -522,10 +522,48 @@ namespace aspect
                                              &sand_transport_coefficient,
                                              &silt_transport_coefficient);
 
+          // generate a combined array for kf kd both onshore and offshore
+          std::vector<double> combined_kd(fastscape_array_size);
+          std::vector<double> combined_kf(fastscape_array_size);
```

Comment:
The changes to compute these two fields are unrelated to the current patch. Is this something you wanted to make in a different PR? If so, would you take these changes here back out?

Reply:
OK. In general, however, we like to let each PR address a single issue. If you find something that is wrong/that can be improved, make that a separate PR instead of making it a part of another PR.